.github/actions/dockerfiles/Dockerfile.blocklist-client.debian

```diff
@@ -32,9 +32,10 @@ RUN apt-get update && \
     apt-get install -y curl && \
     curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg && \
     chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg && \
-    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null && \
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" > /etc/apt/sources.list.d/github-cli.list && \
+    echo "Package: gh\nPin: origin cli.github.com\nPin-Priority: 1001" > /etc/apt/preferences.d/gh-cli && \
     apt-get update && \
-    apt-get install -y gh=2.69.0
+    apt-get install -y gh
```

Comment:
Do we not want a pinned version anymore? I guess we're ensuring (or at least trying to ensure) that the package is from the github repo and signed so maybe it isn't that big of a deal, but it could "randomly break" the workflow if it updates.

Reply:
Ah okay 🫡